package.json

```diff
@@ -200,6 +200,19 @@
           },
           "default": null,
           "description": "Development setting: The PPE environment for testing."
+        },
+        "azure.authenticationLibrary": {
+          "type": "string",
+          "enum": [
+            "ADAL",
+            "MSAL (Preview)"
+          ],
+          "enumDescriptions": [
+            "Azure Active Directory Authentication Library",
+            "Microsoft Authentication Library (Preview)"
+          ],
+          "default": "ADAL",
```

Comment:
I would not specify a default in package.json since we will likely be changing it in the near future. Instead, I would handle the default logic inside our typescript code - which is actually where any A/B framework stuff would go if we end up doing that.